src/EventLoop/Internal/AbstractDriver.php

```diff
@@ -640,6 +640,20 @@ private function createErrorCallback(): void
         };
     }
 
+    private function callbackId(): string
+    {
+        $callbackId = $this->nextId;
+
+        if (\PHP_VERSION_ID >= 80300) {
+            /** @psalm-suppress UndefinedFunction */
+            $this->nextId = \str_increment($this->nextId);
```

Comment:
str_increment should be added to .php-cs-fixer.dist.php

Reply:
Thanks for the hint, I didn't remember we already had a list of functions defined there.